app/views/govuk_publishing_components/components/_layout_for_public.html.erb

```diff
@@ -14,6 +14,12 @@
   show_search = local_assigns.include?(:show_search) ? local_assigns[:show_search] : true
   title ||= "GOV.UK - The best place to find government services and information"
 
+  show_account_layout ||= false
+  account_nav_location ||= nil
```

Comment:
can we document at least this `account_nav_location` flag (if not all of them) in the component yaml? It's not clear which values are allowed and how they affect the output.

Reply:
Alex they should all be documented here [here](https://github.com/alphagov/govuk_publishing_components/blob/cf063218fb57ace309bd057aaac166a0b785e657/app/views/govuk_publishing_components/components/docs/layout_for_public.yml#L78) but to be fair I could probably add better descriptions to be clearer about what they do (it's pretty clear if you look at the component previews though, I think)

Reply:
Have added more detail to the component docs to make it clearer what each option is there for!

Reply:
Thank you! None of them blockers for the release, but good reminder for our future selves.